software/director

```diff
@@ -1 +1 @@
-Subproject commit 39c78e73d63704a66a80bfce19b58a3e140e017a
+Subproject commit 532a3a981c2a978f0578cf06565e08c53fee9902
```

Comment:
Director https://github.com/RobotLocomotion/director/pull/195 has been merged, so this can be updated to master.